examples/contrib/fom/fom_example.py

```diff
@@ -0,0 +1,115 @@
+"""Simple example of implemented figure-of-merits (FOMs)
+
+Numerical test of a few implemented FOMs (mean square error, mean
+absolute error, density standard deviation, density range, blur, and false
+structures) as a function of increasing noise level.
+
+"""
+
+import odl
+import odl.contrib.fom
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Discrete space: discretized functions on the rectangle
+# [-20, 20]^2 with 100 samples per dimension.
+reco_space = odl.uniform_discr(
+    min_pt=[-20, -20], max_pt=[20, 20], shape=[100, 100])
+
+# Create a discrete Shepp-Logan phantom (modified version)
+phantom = odl.phantom.shepp_logan(reco_space, modified=True)
+
+mse = []
+mae = []
+mvd = []
+std_diff = []
+range_diff = []
+blur = []
+false_struct = []
+
+# Create mask for ROI to evaluate blurring and false structures. Arbitrarily
+# chosen as bone in Shepp-Logan phantom.
+mask = (np.asarray(phantom) == 1)
+
+for stddev in np.linspace(0.1, 10, 100):
+    phantom_noisy = phantom + odl.phantom.white_noise(reco_space,
+                                                      stddev=stddev)
+    mse.append(odl.contrib.fom.supervised.
```

Comment:
We shouldn't do that if we don't have to. `import a as b` puts one extra layer of obscurity to names which should be avoided if possible. The `import numpy as np` is an exception since basically everybody does it (including the Numpy people themselves). But in general, renaming imports should be avoided.

In this case, `from odl.contrib import fom` should be sufficient (it's the recommended way of importing `contrib` packages anyway).